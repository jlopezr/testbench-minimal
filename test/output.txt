Ingrese la masa del satélite (kg): Ingrese la distancia del satélite al centro de la Tierra (m): La energía orbital total del satélite es: -2.85e+10 Joules
