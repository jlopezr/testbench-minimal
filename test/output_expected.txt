Ingrese la masa del satélite (kg): 1000
Ingrese la distancia del satélite al centro de la Tierra (m): 7000000
La energía orbital total del satélite es: -2.85e+10 Joules